### DISCUSSION TITLE
Adicionar Funcionalidade Para Checklist de Celular

### DISCUSSION THREAD

**Discussion by jackanilto** [RamonSilva20/mapos#2008] [Ideas]
Boa tarde meus amigos, quero parabenizar pelo projeto e saber de voces se é possivel adicionar um Check list na tela de cadastro da Ordem de serviço por ex:
Display
Carcaça
Bateria
Autofalante
Auricular
Sinal da antena
Etc...
dessa forma seria feito uma checagem na entrada do equipamento
poderia ser em uma TAB 

eu ainda estou tentando aprender a editar algo por aqui 
Obrigado

**Comment by willph** [comment]
Oficina de TV e rádio?

**Comment by andremp15** [comment]
Uma check list ajudaria porém o que mais seria incrível é poder cadastrar equipamento marca número de série senha padrão e assinatura digital top hein fico imaginando. Vou compartilhar um arquivo que eu mesmo fiz no Corel draw, modelo de OS de entrada.
![IMG-20221230-WA0029](https://user-images.githubusercontent.com/117945999/210120081-2b84724f-d118-44df-b655-3afe1fa371b0.jpg)
